pkgs/development/python-modules/verilogae/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  buildPythonPackage,
+}:
+
+buildPythonPackage rec {
+  pname = "verilogae";
+  version = "1.0.0";
+
+  src = fetchurl {
```

Comment:
Please use fetchPypi and see other examples in the repo how to properly fetch the wheel.

Also why do we don't build from source?

Reply:
@SuperSandro2000 I don't mean to be rude but all of this has previously been mentioned. It has become overwhelming especially since there are now multiple reviewers with no clear progress of getting this finalized. I think a PR shouldn't need more than 3 reviewers at one time.